internal/driver/driver_test.go

```diff
@@ -633,27 +684,27 @@ func cpuProfileSmall() *profile.Profile {
 
 	var cpuL = []*profile.Location{
 		{
-			ID:      1000,
+			ID:      1,
```

Comment:
Why did the ID changed?

Reply:
when the profile gets serialized, it generates a fresh set of ID's starting from 1. So when it compares the saved profile to the profile created in this function, the id's would be different and the test would fail.

Reply:
I think we should update the IDs before we compare then. We should keep testing that the profiles with sparse IDs are read well imo.

Reply:
I did some investigating and found that the ID's get regenerated in the fetching step, where it merges profiles from multiple sources. I changed the code to fetch the expected profile as well, so that both profiles have normalized IDs and we dont have to change the IDs of the test profiles.